script/deploy-chime-alpha-demo

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -euxo pipefail
+
+cd $GITHUB_WORKSPACE/demos/serverless
+npm ci
+
+echo "Deploying to alpha stage for canary"
+
+npm run deploy -- -b chime-sdk-demo-canary -s chime-sdk-demo-canary -o chime-sdk-demo-canary -u false -i eu-south-1 -t -l
```

Comment:
If the script is this short, there is no reason to have separate script, you can just put stuff straight to GitHub action

Reply:
I tried it a lot of times everytime deploy command was giving some issue when i tried to run it directly in github action. All the failed attempts https://github.com/aws/amazon-chime-sdk-js/actions/workflows/deploy.yml

Reply:
that just seems you are in the wrong folder no?

Reply:
Is there any use case of us wanting to use this besides the github action? manual deployment? If so, we want to have it as a seperate script @ltrung

Reply:
No these scripts are only used for github actions

Reply:
I tried multiple time to cd into serverless folder but that command was not working.

Reply:
@ltrung Updated such that all the jobs use a single script file and added a new env variable "NAME" based upon which script deploys correct demos.